docs/community-release.md

```diff
@@ -0,0 +1,84 @@
+# Community Release
+
+1. Ensure all necessary PRs merged to master
```

Comment:
Should we modify PR guidelines as well as part of this PR? Need to mention that we would require acceptance tests with each feature PR.